gravitee-apim-definition/gravitee-apim-definition-model/src/main/java/io/gravitee/definition/model/Plan.java

```diff
@@ -162,6 +162,9 @@ public void setStatus(String status) {
 
     @JsonIgnore
     public List<Plugin> getPlugins() {
-        return this.getFlows().stream().map(Flow::getPlugins).flatMap(List::stream).toList();
+        return Optional
+            .ofNullable(this.getFlows())
```

Comment:
How can it be null if the default is an empty list?

Reply:
It happened in the integration tests, maybe when deserializing definition from file.